DataConnectors/Agari/AzureFunctionAgari/run.ps1

```diff
@@ -278,6 +325,12 @@ if (($bpEnabled) -and ($bpToken)) {
 
 #Check if APD is enabled 
 if (($apdEnabled) -and ($apdToken)){
+        #Check to see if start time empty - usally first run, otherwise set the time from the env variable
+        if ($apdlastLog){
+            $startdate = $APDlastLog.substring(1)  
+        } else {
+            $startdate=(Get-Date (get-date).addMinutes(-6) -UFormat "+%Y-%m-%dT%H:%M:00:001Z")
```

Comment:
Same Q as above. Plus again, best to use a singe time reference stored in a variable.

Reply:
ack.